src/m3ninx/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Remove this. Probably just want one license for the whole monorepo.

Reply:
sure thing

Reply:
done